ecms_base/features/custom/ecms_event/ecms_event.info.yml

```diff
@@ -21,12 +21,15 @@ dependencies:
   - 'drupal:user'
   - 'drupal:views'
   - 'ecms_feeds:ecms_feeds'
+  - 'ecms_paragraphs:ecms_paragraphs'
+  - 'ecms_person:ecms_person'
```

Comment:
I was wondering about this as well. I can either remove the paragraph exclusion from the package. So that someone could add a "person additional fields" paragraph to an event body. Or we can keep it this way. Which do you think is best?

Reply:
I feel like they should be decoupled. If someone ended up adding this paragraph type to an event it shouldn't hurt anything, they would simply be adding a label+value text output. My vote is to separate